src/rt/dmain2.d

```diff
@@ -456,9 +456,6 @@ private extern (C) int _d_run_main2(char[][] args, size_t totalArgsLength, MainF
     }
     version (CRuntime_Microsoft)
     {
-        version (LDC)
-            init_msvc();
-
```

Comment:
There's another call a bit later in `rt_init()`; its only job is to set the `std{out,err,in}` pointers.